pkgs/by-name/me/meteor-git/package.nix

```
@@ -0,0 +1,27 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+}:
+
+buildGoModule rec {
+  pname = "meteor-git";
+  version = "0.22.0";
+
+  src = fetchFromGitHub {
+    owner = "stefanlogue";
+    repo = "meteor";
+    rev = "v${version}";
+    hash = "sha256-aY/gOKvcKtOnL4FI2SM339LU4HoWYCq0W9mK2GyMqso=";
+  };
+
+  vendorHash = "sha256-jKd/eJwp5SZvTrP3RN7xT7ibAB0PQondGR3RT+HQXIo=";
+
+  meta = {
+    description = "A highly configurable CLI tool for writing conventional commits ";
```

According to the [recommendations for meta attributes](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes) the description should not start with an article and avoid subjective language.
```suggestion
    description = "Configurable CLI tool for writing conventional commits ";
```